evm/its.js

```diff
@@ -486,14 +553,25 @@ async function processCommand(config, chain, options) {
             printInfo('Trusted chains', trustedChains);
             printInfo('Trusted addresses', trustedAddresses);
 
+            // check if all trusted addresses match ITS address
+            for (const trustedAddress of trustedAddresses) {
+                if (trustedAddress !== interchainTokenServiceAddress) {
+                    printError(
+                        `Error: Trusted address ${trustedAddress} does not match InterchainTokenService address ${interchainTokenServiceAddress}`,
+                    );
+                }
+
+                break;
```

Comment:
Why `break` here? Wouldn't this only check the first address in the array?

Reply:
good catch, this was supposed to be inside the if. Finding one error at a time is sufficient (to avoid printing tons of addresses)

Reply:
got it makes sense